src/go/rdctl/pkg/utils/utils_unix.go

```diff
@@ -0,0 +1,39 @@
+//go:build unix
```

Comment:
The tag should be unnecessary (because the file ends with `_unix.go`).

Reply:
I tried it without... it didn't work unfortunately :disappointed: